.gitlab-ci.yml

```diff
@@ -68,6 +68,8 @@ stages:
   except:
     refs:
       - schedules
+    variables:
+      - $CI_COMMIT_TITLE =~ /^v[0-9.]+$/
```

Comment:
By looking at commit on main, we should probably take into account that the commit title contains the related PR, ex:
```
v3.6.11 (#1126)
```